plugins/modules/remote_management/lxca/lxca_nodes.py

```diff
@@ -20,6 +20,7 @@
   uuid:
     description:
       uuid of device, this is string with length greater than 16.
+    type: str
```

Comment:
all of them are not defined as strings in INPUT_ARG_SPEC, shouldn't we add `type='str'` to every option there?

Reply:
`type='str'` is the default in the argument spec, so it's not really needed. I personally prefer it, but I also know other people prefer not to.

Reply:
@felixfontein ok, thanks for the info, will know